### ISSUE TITLE
6play.fr: detection

### ISSUE THREAD

**Issue by lilliade** [uBlockOrigin/uAssets#11305]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.6play.fr/`

### Category

detection

### Describe the issue

This web site is an official french TV website
when opening a video on site example : 
`https://www.6play.fr/lego-masters-p_21195/emission-4-c_12916855` 
uBO is detected : here is the message in french
"Un bloqueur de publicité empêche la lecture.
Veuillez le désactiver pour démarrer la vidéo."

Thanks in advance for your support

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![2022-01-13 17_51_50-Lego Masters _ Émission 4 11_01_2022- 6play](https://user-images.githubusercontent.com/97690915/149373713-2ab62578-94ad-47c6-91a4-65a391485b92.jpg)
</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.40.8
Firefox: 96
filterset (summary): 
  network: 78267
  cosmetic: 40484
  scriptlet: 16414
  html: 616
listset (total-discarded, last updated): 
  removed: 
    FRA-0: null
  default: 
    ublock-filters: 30740-29, 1h.11m
    ublock-badware: 3662-1, 2d.14m
    ublock-privacy: 183-0, 1h.13m
    ublock-abuse: 76-0, 1h.8m
    ublock-unbreak: 1728-0, 1h.9m
    easylist: 62917-582, 1h.10m
    easyprivacy: 26622-481, 1h.12m
    urlhaus-1: 7413-0, 1h.14m
    plowe-0: 3682-2, 10d.18h.39m
filterset (user): [array of 1 redacted]
trustedset: 
  added: [array of 1 redacted]
  removed: 
    about-scheme
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  launchToReadiness: 439 ms (selfie)
popupPanel: 
  blocked: 219
```
</details>

**Comment by mapx-** [comment]
maybe they are checking for vpn / proxy, I get:

![image](https://user-images.githubusercontent.com/10303732/149375896-0c8bfd24-d5f3-476a-b7a6-b524cd6b3c3c.png)

**Comment by lilliade** [comment]
I suppose the streaming is limited to IP coming from France

**Comment by uBlock-user** [comment]
>  removed: 
    FRA-0: null

Why did you disable the regional list ?

**Comment by iam-py-test** [comment]
I can not reproduce:
![image](https://user-images.githubusercontent.com/84232764/149525040-c0ba09f8-1ab1-40b9-9fc7-6b34663960da.png)
Maybe I'm using a weird VPN it can't detect...

**Comment by Procyon-b** [comment]
Here is my take.
I'm from Belgium (neighboring country) but can't access 6play directly. Using a french proxy, I was able only once to play the video before the site gave me errors. The video is now unavailable (404). I've tried other videos with this proxy (and others), but I always get the same error that the video is not available in my country.
I'll make the same remark as @uBlock-user : why is the regional list "FRA: AdGuard Français" disabled? There are 10 filters targeting `6play.fr` in it.
@lilliade : A first step to find a solution is to enable the list and try again.
